fjord/feedback/views.py

```diff
@@ -44,23 +46,39 @@ def download_firefox(request, template):
 
 
 def thanks(request):
-    if waffle.flag_is_active(request, 'thankyou'):
-        template = 'feedback/thanks.html'
-        try:
-            opinion_id = request.session.get('opinion_id')
-            user_opinion = Response.objects.get(id=opinion_id)
-        except Response.ObjectDoesNotExist:
-            pass
-        else:
-            word_count = len(user_opinion.description.split())
-            if (user_opinion.locale == 'en-US' and
-                    not user_opinion.happy and
-                    word_count >= 7):
-                template = 'feedback/thanks_sad.html'
-    else:
-        template = 'feedback/thanks.html'
+    suggestions = None
+    response = None
 
-    return render(request, template)
+    if waffle.flag_is_active(request, 'thankyou'):
+        response_id = None
+
+        # If the user is an analyzer/admin, then we let them specify
+        # the response_id via the querystring. This makes debugging
+        # the system easier.
+        if ((request.user.is_authenticated()
+             and request.user.has_perm('analytics.can_view_dashboard'))):
```

Comment:
I think you can skip the `request.user.is_authenticated()` part. The anonymous user won't have this permission. Or hey, maybe they will, because it was configured that way.